### PR TITLE
Improve territories rendering, fix #341

### LINKED ISSUE #341: Improve territories rendering
* [x] darker blue for the header
* [x] discuss logo (producer vs. town)
* [x] make it possible to close the embed window
* [ ] order datasets by probability of having data (actually a bug solved so not needed anymore)
* [x] add a call-to-action for town owners
* [x] remove the bubble on maps
* [ ] discuss pertinence of higher levels in the aside
* [ ] propose new titles/descriptions (mail sent to Claire-Marie, no answer yet)
* [x] resize the component

### PR DESCRIPTION
**Pull request by davidbgk**
See https://github.com/etalab/udata/issues/341 for description.

### CODE REVIEW

File: udata/features/territories/models.py [hunk 1]
```diff
@@ -24,6 +27,10 @@ def slug(self):
         return '{territory_id}-{id}'.format(
             territory_id=self.territory.id.replace('/', '-'), id=self.id)
 
+    @property
+    def organization(self):
+        return Organization.objects.get(id=self.organization_id)
```
Comment (vinyll) [inline udata/features/territories/models.py]: Would mongoengine cache this query or would each call to `territory_dataset.organization` make a new request?

File: udata/templates/territories/territory.html [hunk 2]
```diff
@@ -38,6 +40,11 @@
                 </strong>
             </p>
         </div>
+        <a href="mailto?" type="button" class="btn btn-warning btn-follow btn-block btn-sm"
```
Comment (vinyll) [inline udata/templates/territories/territory.html]: `href="mailto?"`?

File: udata/features/territories/models.py (hunk 1 above)
Comment (davidbgk) [inline udata/features/territories/models.py]: You're right, I'll use `@cached_property`.

File: udata/templates/territories/territory.html (hunk 2 above)
Comment (davidbgk) [inline udata/templates/territories/territory.html]: Yes, we have to discuss what to do upon that call-to-action :)

File: js/dataset/display.js [hunk 3]
```diff
@@ -162,10 +162,9 @@ function load_coverage_map() {
         onEachFeature: function(feature, layer) {
             layer.bindPopup(feature.properties.name);
             layer.on('mouseover', function() {
-                layer.openPopup();
-            });
-            layer.on('mouseout', function() {
-                layer.closePopup();
+                if (!$el.data('hide-popup')) {
+                    layer.openPopup();
```
Comment (noirbizarre) [inline js/dataset/display.js]: :+1:

File: udata/features/territories/models.py [hunk 4]
```diff
@@ -24,6 +28,10 @@ def slug(self):
         return '{territory_id}-{id}'.format(
             territory_id=self.territory.id.replace('/', '-'), id=self.id)
 
+    @cached_property
```
Comment (noirbizarre) [inline udata/features/territories/models.py]: +1

File: udata/templates/territories/territory.html [hunk 5]
```diff
@@ -38,6 +40,11 @@
...
+            rel="tooltip" data-placement="left"
```
Comment (noirbizarre) [inline udata/templates/territories/territory.html]: I think it should follow the html compatibility fix: `data-toggle="tooltip'` ;)

File: udata/templates/territories/territory.html (hunk 2 above)
Comment (noirbizarre) [inline udata/templates/territories/territory.html]: Until further discussion, maybe we should redirect to the dataset page.

**Comment by noirbizarre** [comment]
r+wc

File: udata/templates/territories/territory.html (hunk 5 above)
Comment (davidbgk) [inline udata/templates/territories/territory.html]: Yup, we should merge master to dev first ;)

File: udata/templates/territories/territory.html (hunk 2 above)
Comment (davidbgk) [inline udata/templates/territories/territory.html]: Mmh I don't think it's consistent with the wording in that case, here we want the user to notify us that he is interested in participating for his territory. And I thought it was what we discussed during the demo. Did I miss something?

Comment (noirbizarre) [inline udata/templates/territories/territory.html]: Sorry, I didn't see the wording. OK, we will discuss this tomorrow.

File: udata/templates/territories/territory.html (hunk 5 above)
Comment (noirbizarre) [inline udata/templates/territories/territory.html]: OK, I'll merge it tomorrow ;)

**Pull request merged**